### ISSUE TITLE
Fall-through false-positive on cases that contains a throw

### ISSUE THREAD

**Issue by Taluu** [squizlabs/PHP_CodeSniffer#2101]
For the given code :

```php
<?php
//...
switch ($content) {
     case 'foo':
          throw new ...;
     case 'bar':
          // ...
}
```

I have the warning that says that there should be a comment line saying the fall-through is intentional.... But a throw interrupts the case anyway (unless it is caught in the same block, which is not the case here). It seems the sames goes for a `return`.

Adding a `break` or a fall-through comments is over-kill as it will never be reached...

**Comment by Taluu** [comment]
More or less a duplicate of #834. Feel free to close and continue in this issue.

**Comment by Taluu** [comment]
Nevermind, it's because I was in a nested switch which needed the comment / break. Closing.
